src/bots/botRouter.ts

```diff
@@ -74,7 +75,7 @@ export class BotRouter implements IBotRouter {
 
   // eslint-disable-next-line no-warning-comments
   // TODO - get only if bot.isBelowRateLimit() is true
```

Comment:
I want all `eslint-disable-next-line no-warning-comments` comments gone and the all `TODO` comments as well. They should not be in the code base but in `linear` and/or noted in the tsdoc of the method if it concerns the behavior of the method.